lib/DDG/Spice/GetEvents.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::GetEvents;
+#ABSTRACT: Returns events near the user's location.
+
+use DDG::Spice;
+
+#Attribution
+primary_example_queries "events near me";
+description "Upcoming events from GetEvents";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/GetEvents.pm";
+category "entertainment";
+topics "computing","entertainment","food_and_drink","music","science","special_interest","travel";
+
+#Triggers
+triggers startend => 'events near me';
+                                                                                                                                                                                                                                  
+spice to => 'https://api.getevents.co/ddg?lat=$1&lng=$2&timezone=$3' ;
+# Extract the geo location from the result returned by the handler: two real numbers seperated by '/' and after them a time zone also seperated by a '/'.
+spice from => '(?:([-+]?[0-9]*\.?[0-9]+)\/([-+]?[0-9]*\.?[0-9]+))\/(?:([a-zA-Z_]+%2F[a-zA-Z_]+))';
```

Comment:
Actually the current regex is causing some problems for NGINX. Please change it to this:

```
spice from => '([^/]+)/([^/]+)/(.*)';
```